x-pack/test_serverless/functional/test_suites/observability/observability_log_explorer/header_menu.ts

```diff
@@ -159,13 +161,13 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         });
 
         await retry.try(async () => {
-          expect(await PageObjects.observabilityLogExplorer.getQueryBarValue()).not.to.eql(
+          expect(await PageObjects.observabilityLogsExplorer.getQueryBarValue()).not.to.eql(
             '*favicon*'
           );
         });
 
-        // go to log explorer tab
-        await testSubjects.click('logExplorerTab');
+        // go to logs explorer tab
+        await testSubjects.click('logsExplorerTab');
         await PageObjects.svlCommonNavigation.breadcrumbs.expectBreadcrumbExists({
           deepLinkId: 'discover',
```

Comment:
Should we also change `deepLinkId` of observability Logs Explorer app? (around line 175)

Reply:
Changed in https://github.com/elastic/kibana/pull/175711/commits/889e2a71e3e7835c282feb2e33e2a667e68a037e